index.html

```diff
@@ -0,0 +1,235 @@
+                    <div id="work3role"><h5>Back End Dev</h5></div>
+                    <div id="work3counter1"></div>
+                    <div id="work3text"><h5>2015</h5></div>
+                </div>
+                <div class="work3primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work3tags">
+                    <div id="work3tag1"><div id="html3">html</div></div>
+                    <div id="work3tag2"><div id="css3">css</div></div>
+                    <div id="work3tag3"><div id="javascript3">javascript</div></div>
+                </div>
+                <div class="work3Actions">
+                    <div id="work3see-project">
+                        <div><h4>See Project</h4></div>
+                    </div>
+                </div>
+            </div>
+            <div class="work4">
+                <img src="SnapshootPortfolio2.png" alt="">
+               
+                <h2>Multi-Post Stories</h2>
+                <div id="work4frame">
+                    <div id="work4text1"><h5>CANOPY</h5></div>
+                    <div id="work4counter"></div>
+                    <div id="work4role"><h5>Back End Dev</h5></div>
+                    <div id="work4counter1"></div>
+                    <div id="work4text"><h5>2015</h5></div>
+                </div>
+                <div class="work4primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work4tags">
+                    <div id="work4tag1"><div id="html4">html</div></div>
+                    <div id="work4tag2"><div id="css4">css</div></div>
+                    <div id="work4tag3"><div id="javascript4">javascript</div></div>
+                </div>
+                <div class="work4Actions">
+                    <div id="work4see-project">
+                        <div><h4>See Project</h4></div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="About-myself">
+            <section class="top">
+                <div id="primary-heading"><h2>About Myself</h2></div>
+                <div id="primary-paragraph">
+                    <p>Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div id="social-media">
+                    <div id="supporting-text">
+                        <div id="lets-connect1"><h4>LET’S CONNECT</h4></div>
+                    </div>
+                    <div id="frame1">
+                        <ul>
+                            <li><img src="Vector2.png" alt=""></li>
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that you surround your social media links with an `<a>`  tag because they are links and users should be able to use them to get to your profiles.
- [ ] Please apply the changes to everwhere affected in your project.